x-pack/plugins/maps/public/meta.js

```diff
@@ -24,12 +30,34 @@ export async function getDataSources() {
 
   loadingMetaPromise = new Promise(async (resolve, reject) => {
     try {
-      const response = await fetch(`${GIS_API_RELATIVE}/meta`);
-      const metaJson = await response.json();
+      const proxyElasticMapsServiceInMaps = chrome.getInjected('proxyElasticMapsServiceInMaps', false);
+      if (proxyElasticMapsServiceInMaps) {
+        const fullResponse = await fetch(`${GIS_API_RELATIVE}/${EMS_META_PATH}`);
+        emsSources = await fullResponse.json();
+      } else {
+        const emsClient = new EMSClient({
+          language: i18n.getLocale(),
+          kbnVersion: chrome.getInjected('kbnPkgVersion'),
+          manifestServiceUrl: chrome.getInjected('emsManifestServiceUrl'),
+          landingPageUrl: chrome.getInjected('emsLandingPageUrl')
+        });
+        const isEmsEnabled = chrome.getInjected('isEmsEnabled', true);
+        const xpackMapsFeature = xpackInfo.get('features.maps');
+        const licenseId = xpackMapsFeature && xpackMapsFeature.maps && xpackMapsFeature.uid ? xpackMapsFeature.uid :  '';
+
+        const emsResponse = await getEMSResources(emsClient, isEmsEnabled, licenseId, proxyElasticMapsServiceInMaps);
```

Comment:
`const emsResponse = await getEMSResources(emsClient, isEmsEnabled, licenseId, proxyElasticMapsServiceInMaps);` is setting `useProxy` with wrong value. Instead of `proxyElasticMapsServiceInMaps`, should be `!proxyElasticMapsServiceInMaps` or better yet, just `false` since the else statement is only when proxy is not enabled

Reply:
i knew I was going to forget flipping a boolean